File: sample_code/php/pertemuan_3/variable.php
<?php 
// Deklarasi variable
//penulisan standar: [tipe data] [nama variable] 

//contoh penulisan variable di bahasa php
$nama_variable = 'ini string';

//variable dengan tipe data string
$nama_mahasiswa = 'Resita';

//variable dengan tipe data integer
$nilai = 100;

//deklarasi variable dengan tipe data boolean
$boolean = true;
$boolean_false = false;

//deklarasi variable dengan tipe data float / double /decimal
$float = 0.05;

//menampilkan output di browser
echo $nama_mahasiswa;
echo "<br>";
echo $nilai;
echo "<br>";
echo $boolean;
echo "<br>";
echo $boolean_false;

//variable dideklarasikan lagi
$nilai = 98;
echo $nilai;

echo "<br>";
echo $float;
echo "<br>";
//cara cek variable
var_dump($nama_mahasiswa);

File: materi/pertemuan_3/kotlin/variable.kt
fun main () {
    // deklarasi variable
    val nama_variable = "ini string"
    val nama_mahasiswa = "Resita" // gunakan val jika nilai tetap
    var nilai = 100 // gunakan var jika ingin mengubah suatu nilai

    //deklarasi variable
    val boolean_var = true
    val ini_boolean_false = false

    //deklarasi variable tipe data float
    val float_var = 0.05f

    println("""
        $nama_mahasiswa
        $nilai
        $boolean_var
        $float_var
    """.trimIndent()) // gunakan trimIndent dengan tanda """ 3 untuk mempermudah menggunakan tagbreak

    nilai = 98
    println("$nilai")
}